### ISSUE TITLE
Links broken for today's meeting

### ISSUE THREAD

**Issue by hhorak** [fedora-infra/mote#38]
Hi, we had a meeting today (2015-06-18, env-and-stacks) and I wanted to access meeting logs. I got to page http://localhost/meetbot/fedora-meeting-2/2015-06-18/env-and-stacks.2015-06-18-12.00.html which obviously doesn't work.. but even http://meetbot.fedoraproject.org/fedora-meeting-2/2015-06-18/env-and-stacks.2015-06-18-12.00.html shows just;
```
Not Found

The requested URL /meetbot/teams/env-and-stacks/env-and-stacks.2015-06-18-12.00.html was not found on this server.
```

**Comment by Cydrobolt** [comment]
@hhorak hello! Sorry for the inconvenience! We are aware of this issue and have patched it in 029a6ef8be9bc9a9fa04ddd281c1ced588bfa7f3

I am currently rebuilding mote and preparing to deploy it. The issue should be fixed as soon as all the builds finish and we deploy the bugfix, it should be fixed by the end of the day.

Thanks!

**Comment by Cydrobolt** [comment]
@hhorak I've also manually fixed the meeting log for you, as a temporary fix while we deploy the update.

http://meetbot.fedoraproject.org/fedora-meeting-2/2015-06-18/env-and-stacks.2015-06-18-12.00.html should now be working.
